gamepad.js

```diff
@@ -14,6 +14,16 @@ pc.script.create('gamepad', function (context) {
         initialize: function () {
             this.client = context.root.getChildren()[0].script.client;
             this.link = context.root.findByName('camera').script.link;
+            this.tank = context.root.findByName('tank');
+            this.teams = context.root.getChildren()[0].script.teams;
+
+
+            this.client.socket.on('tank.new', function (data) {
```

Comment:
you have `function (` (we have `function(` below; that's consistent with the style elsewhere, if you care